app/test_with_resources.bzl

```diff
@@ -0,0 +1,20 @@
+def test_with_resources(file):
+  '''
+  Genrule for test files.
+
+  Because each databinding library must have a unique package name and manifest, resources must be
+  imported using the proper package name when building with Bazel. This genrule alters those imports
+  in order to keep Gradle building.
+  '''
+  native.genrule(
+    name = "update_" + file[0:-3],
+    srcs = [file[0:-3] + ".kt"],
```

Comment:
Could this just be ``srcs = [file]`` given we know it already has to be a Kotlin file?

Reply:
Fixed!